docs/source/index.mdx

```diff
@@ -29,121 +29,18 @@ Now that you have the plugins installed lets look at a simple example of a Prefl
-              message: Unable to determine the distribution of Kubernetes
-    - nodeResources:
-        checkName: Total CPU Cores in the cluster is 2 or greater
-        outcomes:
-          - fail:
-              when: "sum(cpuCapacity) < 2"
-              message: The cluster must contain at least 2 cores
-          - fail:
-              when: "sum(cpuCapacity) < 4"
-              message: The cluster must contain at least 4 cores
-          - pass:
-              message: There are at least 2 cores in the cluster
-```
-
-Ready to try it out? Copy that YAML into a local file named `preflight.yaml` and run the following command to see the results of this Kubernetes version check Preflight Check.
+Preflight Checks can be executed by your customer before installing your application into their Kubernetes cluster. The checks are defined in a YAML file. Here is an example of a simple Preflight Check that could be used to enforce a minimum Kubernetes version on your cluster. Run the following command on your cluster to see an example Preflight Check.
 
 ```shell
-kubectl preflight preflight.yaml
+kubectl preflight https://preflight.replicated.com
 ```
 
 ### Example Support Bundle
 
-Now lets use similar YAML to define a Support Bundle.
-
-A Support Bundle needs to know what to collect and optionally, what to analyze. This is defined in a YAML file. There's a lot already included in the default collectors. Here is an example of a Support Bundle that collects the version of Kubernetes using similar YAML we wrote for the Preflight Check.
-
-```yaml
-apiVersion: troubleshoot.replicated.com/v1beta1
-kind: SupportBundle
-metadata:
-  name: kubernetes-version
-spec:
-  collectors: []
-  analyzers:
-    - clusterVersion:
-        outcomes:
-          - fail:
-              when: < 1.16.0
-              message: >-
-                The application requires at Kubernetes 1.16.0 or later, and
-                recommends 1.18.0.
-              uri: 'https://kubernetes.io'
-          - warn:
-              when: < 1.18.0
-              message: >-
-                Your cluster meets the minimum version of Kubernetes, but we
-                recommend you update to 1.18.0 or later.
-              uri: 'https://kubernetes.io'
-          - pass:
-              message: >-
-                Your cluster meets the recommended and required versions of
-                Kubernetes.
-```
-
-Lets try it out and see what is collected by our Support Bundle. Save the YAML into a local file named `bundle.yaml` and execute the following command.
+A Support Bundle needs to know what to collect and optionally, what to analyze. This is defined in a YAML file. There's a lot already included in the default collectors. Here is an example of a Support Bundle that collects the version of Kubernetes. Run the following command on your cluster to see an example Support Bundle.
 
 ```shell
-kubectl support-bundle bundle.yaml
+kubectl support-bundle https://troubleshoot.replicated.com
```

Comment:
@GraysonNull Did we get the redirector changed to `https://support-bundle.replicated.com`?

Reply:
It did yes. I've got another commit coming that updates the videos on the homepage and changes the URL for these to support-bundle.

Reply:
@GraysonNull Sorry, maybe I wasn't clear. This PR uses `https://troubleshoot.replicated.com`. It now redirects to the right location on GitHub, but the URL itself still says `troubleshoot` instead of `support-bundle`. Can we change that?

Reply:
Correct. Yes I have another commit I'm working on that will change it to be `support-bundle.replicated.com` instead of troubleshoot.replicated